cmd/quickStart/projectRunner_test.go

```diff
@@ -20,6 +20,8 @@ func TestProjectRunnerTestSuite(t *testing.T) {
 
 var mockAgents = []org.Agent{
 	{AgentIdentifier: "abc"},
+	{AgentIdentifier: "abc"},
+	{AgentIdentifier: "xyz"},
```

Comment:
Without the "distinct" constraint being added above, the tests would fail because they check the contents of r.AgentIdentifiers. Adding distinct makes the tests pass, thus asserting the behavioral change here didn't require an additional test.